CHANGELOG.md

```diff
@@ -7,13 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 **Note**: Since GDK for Unreal v0.10.0, the changelog is published in both English and Chinese. The Chinese version of each changelog is shown after its English version.<br>
 **注意**：自虚幻引擎开发套件 v0.10.0 版本起，其日志提供中英文两个版本。每个日志的中文版本都置于英文版本之后。
 
+
 ## [`x.y.z`] - Unreleased
 
 ### Breaking changes:
 - SpatialSwitchHasAuthority now respects World's version of IsServer which assumes server status when NetDriver is null.
 
 ### Features:
-- Gameplay Debugger now supports multi-worker environments.
```

Comment:
where this go? In 0.14?

Reply:
Yeah it's in 0.14 so don't want a duplicate.